portable/IAR/ARM_CM55_NTZ/non_secure/port.c

```diff
@@ -1775,7 +1775,7 @@ void vPortEndScheduler( void ) /* PRIVILEGED_FUNCTION */
     void vPortStoreTaskMPUSettings( xMPU_SETTINGS * xMPUSettings,
                                     const struct xMEMORY_REGION * const xRegions,
                                     StackType_t * pxBottomOfStack,
-                                    uint32_t ulStackDepth )
+                                    vPortStoreTaskMPUSettings ulStackDepth )
```

Comment:
This has to be replaced with configSTACK_DEPTH_TYPE

Reply:
Sharp eyes! Fixed.